debezium-quarkus-outbox/integration-tests/src/test/java/io/debezium/outbox/quarkus/it/OutboxTest.java

```diff
@@ -63,7 +63,7 @@ public void testOutboxEntityMetamodelExists() throws Exception {
     public void firedEventGetsPersistedInOutboxTable() {
         myService.doSomething();
 
-        Query q = entityManager.createNativeQuery("SELECT CAST(id as varchar), aggregateId, aggregateType, type, timestamp, payload FROM OutboxEvent");
+        Query q = entityManager.createNativeQuery("SELECT CAST(id as varchar), aggregateId, aggregateType, type, timestamp, payload, openTracingSpan FROM OutboxEvent");
```

Comment:
I'd suggest just `tracingSpan`. Decouples from any changes by moving to OpenTelemetry later on.